src/new-bot/commands/expertise/update.js

```diff
@@ -36,16 +39,143 @@ function updateExpertise({userName, expertise, newValues}) {
   });
 }
 
-const updateMissingHandler = createMatcher({match: 'missing'}, (_, {user}) => {
-  return `> update missing for ${user.name} (coming soon)`;
-});
+function updateExpertiseDialog({
```

Comment:
(Think about how series of dialogs might be authored, in general, because I think this represents a common use-case)